### PR TITLE
Update the vulnerability detector default configuration block with missing providers

### PR DESCRIPTION
**Pull request by vcerenu**
This PR adds SUSE to the vulnerability detector configuration.
Closes https://github.com/wazuh/wazuh-puppet/issues/728

### CODE REVIEW

**Review by davidcr01 — APPROVED** [review]

File: manifests/params_manager.pp [hunk 1]
```diff
@@ -178,14 +178,13 @@
       $vulnerability_detector_provider_debian                 = 'yes'
       $vulnerability_detector_provider_debian_enabled         = 'no'
       $vulnerability_detector_provider_debian_os              = ['wheezy',
-        'stretch',
```
Comment (teddytpc1) [inline manifests/params_manager.pp:181]: For `canonical`, `focal` and `jammy` are missing (line 173)

File: manifests/params_manager.pp [hunk 2]
```diff
@@ -178,14 +178,13 @@
...
         'jessie',
```
Comment (teddytpc1) [inline manifests/params_manager.pp:181]: For `debian`, the correct values are `buster` and `bullseye`.

**Review by teddytpc1 — CHANGES_REQUESTED** [review]

File: manifests/params_manager.pp (hunk 1 above)
Comment (vcerenu) [reply manifests/params_manager.pp:181]: dome

File: manifests/params_manager.pp (hunk 2 above)
Comment (vcerenu) [reply manifests/params_manager.pp]: done

**Pull request closed without merging**